morpho5/docs/meshgen.md

```diff
@@ -0,0 +1,96 @@
+[comment]: # (Meshgen module help)
+[version]: # (0.5)
+
+# Meshgen
+[tagdelaunay]: # (delaunay)
+
+The `meshgen` module is used to create `Mesh` objects corresponding to a specified domain. It provides the `MeshGen` class to perform the meshing, which are created with the following arguments:
+
+    MeshGen(domain, boundingbox)
+
+Domains are specified by a scalar function that is positive in the region to be meshed and locally smooth. For example, to mesh the unit disk:
+
+    var dom = fn (x) -(x[0]^2+x[1]^2-1)
+
+A `MeshGen` object is then created and then used to build the `Mesh` like this:
+
+    var mg = MeshGen(dom, [-1..1:0.2, -1..1:0.2])
+    var m = mg.build()
+
+A bounding box for the mesh must be specified as a `List` of `Range` objects, one for each dimension. The increment on each `Range` gives an approximate scale for the size of elements generated.
+
+To facilitate convenient creation of domains, a `Domain` class is provided that provides set operations `union`, `intersection` and `difference`.
+
+`MeshGen` accepts a number of optional arguments:
+
+* `weight` A scalar weight function that controls mesh density.
+* `quiet` Set to `true` to suppress  `MeshGen` output.
+* `method` a list of options that controls the method used.
+
+Some method choices that are available include:
+
+* `"FixedStepSize"` Use a fixed step size in optimization.
+* `"StartGrid"` Start from a regular grid of points (the default).
+* `"StartRandom"` Start from a randomly generated collection of points.
+
+There are also a number of properties of a `MeshGen` object that can be set prior to calling `build` to control the operation of the mesh generation:
+
+* `stepsize`, `steplimit` Stepsize used internally by the `Optimizer`
+* `fscale` an internal "pressure"
+* `ttol` how far the vertices are allowed to move before retriangulation
+* `etol` energy tolerance for optimization problem
+
+`MeshGen` picks default values that cover a reasonable range of uses.
+
+[showsubtopics]: # (subtopics)
+
+## Domain 
+[tagdomain]: # (domain)
+
+The `Domain` class is used to conveniently build a
```

Comment:
Is the last part of the sentence missing?

Reply:
Great catch there! Will fix later...